doc/mfserv_miscellaneous.md

```diff
@@ -154,7 +154,7 @@ If you need to execute your `cron` command in the Metwork context, you should us
 {% raw %}
 {{MFSERV_HOME}}/bin/cronwrap.sh --lock --low "find {{MFMODULE_RUNTIME_HOME}}/var/archive/ -type f -mtime +5 -exec rm -f {} \;" >/dev/null 2>&1
 
-{{MFSERV_HOME}}/bin/cronwrap.sh -- plugin_wrapper [your_plugin_name]  [your_sh_command] >{{MFMODULE_RUNTIME_HOME}}/log/[your_log_filename] 2>&1
+{{MFSERV_HOME}}/bin/cronwrap.sh -- plugin_wrapper [your_plugin_name]  [your_sh_command] >[your_log_filename] 2>&1
```

Comment:
this doc has to be fixed as it does not use --log-capture-to

Reply:
Done